src/Information/InformationItem.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  ListItem,
+  OrderedList,
+  Text,
+  UnorderedList
+} from '@chakra-ui/react';
+import React from 'react';
+
+interface AccordionItemProps {
+  title: string;
+  index: number;
+  isOpen: number | null;
+  onClick: (index: number) => void;
+}
+
+const InformationItem: React.FC<AccordionItemProps> = ({
+  title,
+  index,
+  isOpen,
+  onClick
+}) => (
+  <AccordionItem>
+    <h2>
+      <AccordionButton
+        onClick={() => {
+          onClick(index);
+        }}
+      >
+        <span>{title}</span>
+        <AccordionIcon />
+      </AccordionButton>
+    </h2>
+    <AccordionPanel pb={4} display={isOpen === index ? 'block' : 'none'}>
+      <Box maxHeight="200px" overflowY="auto">
+        {index === 1 && (
+          <Text>
+            This application is developed by:
+            <OrderedList>
+              <ListItem>Ana Diru</ListItem>
+              <ListItem>Anthony Zigerelli</ListItem>
+              <ListItem>Avinaash Ghansam</ListItem>
+              <ListItem>David Casey</ListItem>
+            </OrderedList>
+          </Text>
+        )}
```

Comment:
Let's just take the accordion content as `children: React.ReactNode`. Right now this is pretty coupled to the information.

Reply:
To clarify, the parent should provide this content through a children prop

Reply:
change made to reflect this